Dockerfile.assets_builder

```diff
@@ -4,14 +4,11 @@ FROM elifesciences/journal_composer:${image_tag} AS composer
 FROM elifesciences/journal_npm:${image_tag} as npm
 FROM node:${node_version}
 
-# This is a temporary solution while we migrate to a more recent version of nodejs
-RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
-
 RUN apt-get update && apt-get install --no-install-recommends -y \
     libvips \
     && rm -rf /var/lib/apt/lists/*
 
-COPY --from=npm /node_modules/ node_modules/
+COPY --from=npm /usr/app/node_modules/ node_modules/
```

Comment:
This seems to be working for me but I'm not sure why. Why was this change necessary?

Reply:
The WORKDIR parameter has been added here: https://github.com/elifesciences/journal/pull/1782/files/65815f0f0514c41b212608ccfeacaeb696930497#diff-d9f6088823a6af2ad6b7eb29ffe019599e6cf6cc9221979578e573b8fd1af65cR8

Reply:
In which case, we can replicate the current behaviour with setting `WORKDIR` to be the expected default?:

```
WORKDIR /
```

Reply:
I tested `WORKDIR /` and it's not working. While building the containers we get the error: `npm ERR! Tracker "idealTree" already exists` 

ref:  https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fjournal/detail/PR-1782/15/pipeline